src/github/push.test.ts

```diff
@@ -65,7 +65,10 @@ describe("PushWebhookHandler", ()=>{
 			repository: {} as GitHubRepository, //force it as not required in test
 			commits: [{
 				id: "commit-1",
-				message: "ARC-0001 some commit message"
+				message: "ARC-0001 some commit message",
+				added: [],
+				modified: [],
+				removed: []
```

Comment:
Had to add these three fields from the official GitHub push webhook type to make the test `push.test.ts` pass. It seems that the feature flag wasn't covered in that particular test 😬.